Gems/WarehouseAutomation/Code/Source/ProximitySensor/ROS2ProximitySensor.h

```diff
@@ -13,11 +13,11 @@
 #include <rclcpp/publisher.hpp>
 #include <std_msgs/msg/bool.hpp>
```

Comment:
Discuss with @michalpelka about making this ROS-independent, as these systems are typically not handled by ROS.

Reply:
Made the proximity sensor ROS-independent.